nurse/api.py

```diff
@@ -40,6 +45,10 @@ def constructor_with_dependency_injection(self, *args, **kwargs):
 def serve(user_class, name=None) -> None:
     """
     Add an instance of a user-defined class to Nurse's services catalog.
+    Given user_class is registered in the catalogue with its class name as key by default.
```

Comment:
By default, a dependency is register for its concrete type, but an interface can be provided.

Reply:
Plutôt registered* non ?

Reply:
Bien vu !